wiki/Contests/M2P/en.md

```diff
@@ -0,0 +1,103 @@
+---
+tags:
+  - m2p
+  - m2p#1
+  - m2p#2
+  - m2p#3
+  - m2p 1
+  - m2p 2
+  - m2p 3
+  - m2p: taiko
+---
+
+# Mapping to Pander: Taiko
+
+![contest banner](img/banner.jpg "Mapping to Pander: Taiko")
+
+**Mapping to Pander: Taiko** (*M2P*) is an osu!taiko mapping contest series with two primary features: there is only one judge per iteration, and contestants may map any song.
+
+The goal this contest is simple: Contestants must create a map that panders best to the chosen judge for the contest. The judge will not be given any criteria; it will be up to them how they score and rank the entries, either by making their own criteria, or simply sorting them from best to worst.
+
+Each iteration of the contest will have a different judge, with different tastes and criteria from the last.
+
+## Schedules
+
+Iterations usually happen every 1 or 2 months, with each iteration's schedule going as follows:
+
+| Event | Duration |
+| --: | :-- |
+| Beatmapping | 2 weeks |
+| Screening | Tentative[^schedules-screening] |
+| Judging | 1 or 2 weeks[^schedules-judging] |
+| Results | 1 day |
+
+## Ruleset
+
+Alongside the Judging Criteria that each judge will provide, there are some rules that all contestants must follow:
+
+- **All entries must be submitted as `.osz` files.** This is to ensure that the judges are able to download and test the entries without hassle.
+- **All entries must be osu!taiko beatmaps.**
+- **Submissions must include only ONE difficulty.** (unless stated otherwise by the judge.)
+- **Songs must not exceed 6 minutes of length.** There will a bit of leeway in cases where the song is really close to the 6 minute mark.
+- **All submissions must be rankable.** No beatmap submitted must be intentionally unrankable. Mistakes happen and points will be deducted for them, but this is a regular mapping contest, not Aspire. (You're allowed to ignore spread rules for the sake of this contest.)
+- **Submissions must NOT be uploaded/shared with others before judging and results are announced.** Sharing your entry publicly by any means before the results are announced will get your entry disqualified, no questions asked.
```

Comment:
if submissions are not anonymous, what about the cases where a mapper hints the judge their identity (say, via a file name or metadata)?

Reply:
submissions are not anonymous only to me, I anonymize them for the judge. the anon tool would clear out all tags/source and use an anon entry name as diffname (which are 2 random words)

if someone alters the artist/title or anything else to be potentially revealing their identity, the entry will be disqualified